utils/lib/flatten.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+
+
+def flatten(ports):
+    """Convert port + width into individual pins.
```

Comment:
Please add arguments docstring and be specific that port and (port, width) are both acceptable formats.

Reply:
Done.